cmd/populator/modules.go

```diff
@@ -19,6 +19,13 @@ end:
 
 func init() {
     modules = []Module{
+		{
+			name:        "liquibase-aws-extension",
+			category:    Extension,
+			owner:       "liquibase",
+			repo:        "liquibase-aws-extension",
+			artifactory: Maven{},
```

Comment:
This looks incorrect. The liquibase-aws-extension downloads below are from Github.